modules/quotes.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+# Configurable snippets
+module Quotes
+  extend Discordrb::Commands::CommandContainer
+
+  command :grab, {
+    help_available: true,
+    usage: '.grab @user',
+    min_args: 1,
+    max_args: 1,
+  } do |event, target|
+    # TODO make this nicer. Feels like a hack.
+    target_id = Integer(target[3..-2])
+    event.channel.history(5, event.message.id).each { |m|
+      if (m.author.id == target_id) and (not m.from_bot?)
+        Quote.create(server_id: event.server.id,
+                     user_id: m.author.id,
+                     text: m.content)
+        embed t('quotes.grab.success')
+        return
+      end
+    }
+    embed t('quotes.grab.failure', target)
+  end
+
+  command :listquotes, {
+    help_available: true,
+    usage: '.listquotes @user',
+    min_args: 1,
+    max_args: 1,
+  } do |event, target|
+    target_id = Integer(target[3..-2])
+    quotes = Quote.where(server_id: event.server.id,
+                        user_id: target_id)
+    quote_list = ""
+    quotes.find_each do |q|
+      quote_list += "id: #{q.id}\n#{q.text}\n"
+    end
+    if quote_list != "" then embed quote_list else embed t('quotes.list.empty', target) end
```

Comment:
check for `quotes.size` instead of the resulting string. also try to split if statements into multiple lines